artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/InstallAbstract.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.activemq.artemis.cli.commands;
+
+import java.io.File;
+
+import io.airlift.airline.Arguments;
+import io.airlift.airline.Option;
+
+public class InstallAbstract extends InputAbstract {
+
+   @Arguments(description = "The instance directory to hold the broker's configuration and data.  Path must be writable.", required = true)
```

Comment:
Nit: inconsistent about 1 or 2 spaces after `.`

Reply:
nice one...

(It was wrong before, as I moved it from ./create.. but still wrong.. thanks!